pkg/cmd/run.go

```diff
@@ -809,10 +817,19 @@ func loadPropertyFile(fileName string) (*properties.Properties, error) {
 	return p, nil
 }
 
-func escapePropertyFileItem(item string) string {
-	item = strings.ReplaceAll(item, `=`, `\=`)
-	item = strings.ReplaceAll(item, `:`, `\:`)
-	return item
+func toPropertyEntry(props *properties.Properties, key string) (string, error) {
+	value, _ := props.Get(key)
```

Comment:
I think we can instead provide that `value` variable as a input variable of the fuction. You only use `props` to extract this value AFAICS

Reply:
Now I've seen you already did it this way in the other function in `common.go`. I'd suggest just using this one then.